### PR TITLE
Restructure to flask's blueprint scafolding and implement  scorecard

### PR DESCRIPTION
**Pull request by kimpaller**
https://flask.palletsprojects.com/en/2.2.x/blueprints/

### CODE REVIEW

**Comment by tfcollins** [comment]
Have a instance running with this code?

**Comment by kimpaller** [comment]
> Have a instance running with this code?

Our current constellation deployment is running these codes.

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,8 @@ USER root
 WORKDIR /app
 ADD . /app
 RUN pip install -r requirements.txt
-RUN cd telemetry_src && python setup.py install; cd ..
+RUN git clone -b artifacts-support https://github.com/sdgtt/telemetry.git
```
Comment (tfcollins) [inline Dockerfile]: can we get this branch merged into main before things diverge too much

File: app/models/boot_tests.py [hunk 2]
```diff
@@ -56,6 +60,11 @@ def __initialize_fields(self):
                         "pytest_failures",
                         "pytest_skipped",
                         "pytest_tests",
+                        "matlab_errors",
+                        "matlab_errors",
```
Comment (tfcollins) [inline app/models/boot_tests.py]: duplicated key

**Review by tfcollins — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (kimpaller) [reply Dockerfile]: clone  "telemetry:master" instead as telemetry:artifacts-support" was already merged to master https://github.com/sdgtt/telemetry/pull/28

https://github.com/sdgtt/constellation/commit/f02743275ed0c1c14adc1444ce70b52198a8ab12

File: app/models/boot_tests.py (hunk 2 above)
Comment (kimpaller) [reply app/models/boot_tests.py]: Removed duplicate
https://github.com/sdgtt/constellation/commit/7b9514b7f68b366792afb162b06f414248b4d274

**Review by cseptimo — APPROVED** [review]

**Pull request merged**